packages/base/src/state/core.rs

```diff
@@ -87,7 +99,7 @@ pub struct UnbondBatch {
     pub slashing_effect: Option<Decimal>,
     pub unbonded_amount: Option<Uint128>,
     pub withdrawed_amount: Option<Uint128>,
-    pub created: u64,
+    pub creation_details: UnbondBatchStatusTimestamps,
```

Comment:
please rename field into something that represents the contents. kinda `status_ts` or `status_timestamps`